src/FluentHub/UserControls/Blocks/FileNavigationBlock.xaml

```diff
@@ -65,12 +65,6 @@
                             FontWeight="SemiBold"
                             Style="{ThemeResource BodyTextBlockStyle}"
                             Text="{x:Bind ViewModel.ContextViewModel.Repository.HeadRefsCount, Mode=OneWay}" />
-
-                        <TextBlock
-                            VerticalAlignment="Center"
-                            Foreground="{ThemeResource ApplicationSecondaryForegroundThemeBrush}"
-                            Style="{ThemeResource BodyTextBlockStyle}"
-                            Text="branches" />
                     </StackPanel>
```

Comment:
if you do liek this, the page shows only "number"

Reply:
it should be like that.
In grammar:
One thing: no `s` (1 branch)
Multiple things or no things: `s` (0 branches/4 branches)
Unless we code it to display different text for different amounts of branches/tags it would be gramatically incorrect.
- But thats complex and is too big on a smaller device. its better without.

Reply:
thanks for that info.

Reply:
@DeveloperWOW64 Cannot be `branch(es)` and `tag(s)`?

Reply:
> @DeveloperWOW64 Cannot be `branch(es)` and `tag(s)`?

doesn't look professional and again, looks bad on smaller devices